requirements_rtd.txt

```diff
@@ -1,12 +1,17 @@
 # sphinx-gallery
 sphinx-gallery
 
-# install spikeinterface locally to use autodoc
--e git+https://github.com/SpikeInterface/spikeextractors.git@master#egg=spikeextractors --install-option='--src modules'
--e git+https://github.com/SpikeInterface/spiketoolkit.git@master#egg=spiketoolkit --install-option='--src modules'
--e git+https://github.com/SpikeInterface/spikesorters.git@master#egg=spikesorters --install-option='--src modules'
--e git+https://github.com/SpikeInterface/spikecomparison.git@master#egg=spikecomparison --install-option='--src modules'
--e git+https://github.com/SpikeInterface/spikewidgets.git@master#egg=spikewidgets --install-option='--src modules'
+spikeextractors>=0.5.6
+spiketoolkit>=0.4.2
+spikesorters>=0.1.4
+spikewidgets>=0.2.3
+spikecomparison>=0.1.0
+
```

Comment:
It would be better to avoid to change at two places to make 2 lines in readthedoc.yml.
I don't known if it is possible to have 2 requirement files.
So we could use the requirement.txt + requirement_rtd.txt

Reply:
I think that in rtd you can specify only one requirement file

Reply:
I try a test here https://github.com/SpikeInterface/spikeinterface/pull/17